rootfs/etc/services.d/home-assistant/run

```diff
@@ -3,11 +3,20 @@
 # Start Home Assistant service
 # ==============================================================================
 
+# shellcheck shell=bash
+
 cd /config || bashio::exit.nok "Can't find config folder!"
 
 # Enable mimalloc for Home Assistant Core, unless disabled
 if [[ -z "${DISABLE_JEMALLOC+x}" ]]; then
   export LD_PRELOAD="/usr/local/lib/libjemalloc.so.2"
   export MALLOC_CONF="background_thread:true,metadata_thp:auto,dirty_decay_ms:20000,muzzy_decay_ms:20000"
 fi
-exec python3 -m homeassistant --config /config
+
+debug_args=()
+if bashio::var.true "${PYTHON_DEBUG:-"false"}"; then
+  debug_args=(-m debugpy --listen 0.0.0.0:5678)
+  bashio::log.info "Enabling Python debug server on port 5678"
+fi
+
+exec python3 "${debug_args[@]}" -m homeassistant --config /config
```

Comment:
Debugpy is available as integration. Why add it here?

<https://www.home-assistant.io/integrations/debugpy>

Reply:
Oh wow, I also agree. Thanks for pointing out the integration to me.